python-sdk/tutorials/automl-with-azureml/forecasting-backtest-single-model/pipeline_helper.py

```diff
@@ -71,6 +71,7 @@ def get_backtest_pipeline(
     run_config = RunConfiguration()
     run_config.docker.use_docker = True
     run_config.environment = env
+    run_config.environment_variables["AUTOML_IGNORE_PACKAGE_VERSION_INCOMPATIBILITIES"] = "True"
```

Comment:
Can we also use different way to get environment?
from azureml.core import Environment

env = Environment.get(experiment.workspace, name='AzureML-AutoML', label="prod")

I think, we also will need to add 
ParallelRunConfig(
<...>
environment_variables={"AUTOML_IGNORE_PACKAGE_VERSION_INCOMPATIBILITIES": True}
)

Reply:
I have unified the method as MM/HTS. For the env retrieving part. We might want to keep as it as there might be some GPU customers